src/services/api.js

```
@@ -566,161 +566,68 @@ export async function updateFlexiRecord(sectionid, scoutid, flexirecordid, colum
+    
+    if (!response.ok) {
+      logger.warn(`Failed to fetch members for section ${section.sectionid}: ${response.status} ${response.statusText}`);
+      continue;
+    }
+    
+    const data = await response.json();
+    if (data.items) {
+      // Process members and handle deduplication
+      data.items.forEach(member => {
+        if (member && member.scoutid) {
+          const scoutId = member.scoutid;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)